permutations.r

```diff
@@ -1,22 +1,49 @@
-Packages <- c("gtools", "readr", "tibble", "dplyr")
 
-lapply(Packages, library, character.only = TRUE)
+################################################################################
+
+# WHAT IS THIS SCRIPT FOR?
+
+# OH GOD WHAT IS ANYTHING FOR?
+
+################################################################################
+
+packages <- c("gtools", "readr", "tibble", "dplyr")
+
+lapply(packages, library, character.only = TRUE)
+
+
```

Comment:
You'll notice I've added just a ton of white space and line breaks. I highly recommend adding line breaks to make the code more readable.

I also suggest following the tidyverse recommendations for white space in lines of code. It keeps the style consistent and makes things more readable.